### PR TITLE
added form

### PR DESCRIPTION
**Pull request by Shevtsooov**
- [DEMO LINK](https://Shevtsooov.github.io/layout_html-form/)
- [TEST REPORT LINK](https://Shevtsooov.github.io/layout_html-form/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,13 @@
 /* styles go here */
```
Comment (naziamoff) [inline src/style.css]: Unnecessary comments should be removed

File: src/index.html [hunk 2]
```diff
@@ -11,7 +11,147 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post"
+    >
```
Comment (naziamoff) [inline src/index.html]: looks easier to read this way, doesn't it?
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
      method="post"
    >
```

File: src/index.html [hunk 3]
```diff
@@ -11,7 +11,147 @@
...
+            autocomplete="off"
+          >
+        </label>
+        <label class="label">
+          Name:
+          <input
+          type="text"
+          name="Name"
+          autocomplete="off"
+          required
```
Comment (naziamoff) [inline src/index.html]: [checklist p.2](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)

**Review by naziamoff — APPROVED** [review]
LGTM 👍 . 
But I see you have issues with indentation - make sure to go through [the checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md) - you will better understand how is it supposed to be formatted.

**Review by naziamoff — CHANGES_REQUESTED** [review]
also, it seems you did not add your readme file to the Pull Request. This is a must, let's do it

**Comment by Shevtsooov** [comment]
I see, working on it!

**Review by naziamoff — CHANGES_REQUESTED** [review]
seems you did not deploy 👁️ 
Let's run `npm run deploy` and add test report/demo links to the PR description. Just copy that from your readme file and add to the description

**Comment by Shevtsooov** [comment]
Oh, really, I forgot about that step..

File: src/index.html [hunk 4]
```diff
@@ -11,7 +11,148 @@
...
+        </label>
+        <label class="label">
+          Password:
+          <input
+          type="password"
+          name="password"
+          minlength="6"
+          maxlength="32"
+          >
+        </label>
```
Comment (naziamoff) [inline src/index.html]: [checklist p.2](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)

I have already submitted this comment but it seems you did not fix it in all the places. Please if I leave some code-style comments - make sure to double-check and fix them in all the usage places.

**Review by naziamoff — APPROVED** [review]
looks better now!

One thing - placeholder should not change font styles on focus, only input text

File: src/index.html (hunk 4 above)
Comment (Shevtsooov) [reply src/index.html]: Could you elaborate on the issue with the placeholder, please?

**Pull request closed without merging**